README.md

```diff
@@ -13,6 +13,16 @@
 
 </div>
 
+# Announcement 📣
+
+Amplitude is introducing a new [TypeScript SDK for Browser](https://www.npmjs.com/package/@amplitude/analytics-browser) [BETA]. This new SDK provides improved developer experience, helps users instrument data more seamlessly and provide more control over data being instrumented using custom plugins.
+
+To learn more about the new SDK, here are some useful links:
+
+NPM: https://www.npmjs.com/package/@amplitude/analytics-browser
```

Comment:
Need bullet point?

Reply:
ohh that looked differently from what i posted in issues lol. thanks for catching this. fixed!